src/commands/chains.ts

```diff
@@ -1,14 +1,17 @@
 import type { Arguments, CommandBuilder } from "yargs";
+import { GlobalOptions } from "../cli.js";
 import { getChains } from "../utils.js";
+import type yargs from "yargs";
 
-export type Options = {
+export type Options = GlobalOptions & {
```

Comment:
You can probably just do a plain type alias here. Not sure why I left that "empty" before?